### PR TITLE
Add MariaDB healthcheck.sh script

### PR DESCRIPTION
**Pull request by alexandrevicenzi**
The script can run many tests as well be used in the container healthcheck instruction to check health status.

### CODE REVIEW

**Comment by alexandrevicenzi** [comment]
This is related to https://github.com/SUSE/BCI-tests/issues/405

File: src/bci_build/package/appcontainers.py [hunk 1]
```diff
@@ -130,6 +130,20 @@
 ]
 
 _MARIADB_ENTRYPOINT = (Path(__file__).parent / "mariadb" / "entrypoint.sh").read_bytes()
+_MARIADB_HEALTHCHECK = (Path(__file__).parent / "mariadb" / "healthcheck.sh").read_bytes()
+_MARIADB_GOSU = b"""#!/usr/bin/env bash
```
Comment (dirkmueller) [inline src/bci_build/package/appcontainers.py:134]: why /usr/bin/env bash here when you hardcode /bin/bash later? you could use the same in both, and I think env is unnecessary here. we knwo what the location of bash is in SLE BCI containers

File: src/bci_build/package/appcontainers.py [hunk 2]
```diff
@@ -130,6 +130,20 @@
...
+
+u=$1
+shift
+opts=$*
```
Comment (dirkmueller) [inline src/bci_build/package/appcontainers.py:138]: this breaks arguments with spaces. suggest to delete this line

File: src/bci_build/package/appcontainers.py [hunk 3]
```diff
@@ -130,6 +130,20 @@
...
+
+if ! id -u $u > /dev/null 2>&1; then
+    echo "Invalid user: $u"
+    exit 1
+fi
+
+setpriv --reuid=$u --regid=$u --clear-groups -- /bin/bash $opts
```
Comment (dirkmueller) [inline src/bci_build/package/appcontainers.py:145]:
```suggestion
setpriv --reuid=$u --regid=$u --clear-groups -- /bin/bash "$@"
```

**Review by dirkmueller — APPROVED** [review]

File: src/bci_build/package/appcontainers.py [hunk 4]
```diff
@@ -163,6 +178,8 @@
             entrypoint=["docker-entrypoint.sh"],
             extra_files={
                 "docker-entrypoint.sh": _MARIADB_ENTRYPOINT,
+                "healthcheck.sh": _MARIADB_HEALTHCHECK,
+                "gosu": _MARIADB_GOSU,
```
Comment (tianon) [inline src/bci_build/package/appcontainers.py:182]: This is _not_ `gosu`, nor is it even properly "`gosu` compatible" -- please give the binary in the image a different name.

(If full `gosu` compatibility is desired, please see https://github.com/tianon/gosu/pull/143 which has been successfully tested against `gosu`'s test suite for all the parsing edge cases I'm aware of and maintain a test suite for.)

Comment (tianon) [reply src/bci_build/package/appcontainers.py:182]: Thank you! ❤️
